main/audit_database.go

```diff
@@ -0,0 +1,127 @@
+package main
```

Comment:
adds a subcommand,`auditDatabase`, which looks through a database directory and counts the size of all all the keys and values, categorizing each by key, and then prints a summary. Helped me figure out was going wrong with our database bloat and seems independently useful.